test/functional/test-error.js

```diff
@@ -464,7 +464,11 @@ describe('reportErrorToServer', () => {
     const e = user().createError('123');
     const data = getErrorReportData(undefined, undefined, undefined, undefined,
         e, true);
-    expect(data.exps).to.equal('test-exp=1,disabled-exp=0');
+    if (data.exps.indexOf('inabox-rov=1') == -1 {
```

Comment:
Branching like this feels weird in a test. What causes the test to have `inabox-rov=1` set in one case but unset in another? Is the test non-hermetic?

Reply:
Not sure. Was going off of https://github.com/ampproject/amphtml/pull/14338/files which expects it to equal 'inabox-rov=1,test-exp=1,disabled-exp=0' but also that PR's Travis failure in which at least one test has it still equalling 'test-exp=1,disabled-exp=0'
Looks like this originated with https://github.com/ampproject/amphtml/pull/13825/files but I'm puzzled as to why it's only happening now, when 13825 is 4 weeks old.

Reply:
I would prefer that this test be deterministic and give the same values each time. Barring that, checking explicitly for the individual values would be better than branching.

Reply:
Ok. Given that I just created this to get rid of redness, and 14352 achieves that, let's let this one sit until @bradfrizzell returns from vacation.